### PR TITLE
Docs/alias

### PR DESCRIPTION
**Pull request by ajesh-cc**
Please review the new content added to cover “Alias” feature.

-Ajesh

### CODE REVIEW

File: docs/features/deploying/manage-projects.md [hunk 1]
```diff
@@ -2,6 +2,40 @@
 title: Manage Projects
 ---
 
+## Hostname / Alias
```
Comment (Disturbing) [inline docs/features/deploying/manage-projects.md]: No need to call this publically as Alias, just Setting service hostname.

File: docs/features/deploying/manage-projects.md [hunk 2]
```diff
@@ -2,6 +2,40 @@
...
+
+The **Alias** feature gives the ability to add the same KintoBlock more than once in a single project by changing the hostname. The changes made to the original KintoBlock will be applied to all the aliases.
```
Comment (Disturbing) [inline docs/features/deploying/manage-projects.md]: Remove all alias words.

File: docs/features/deploying/manage-projects.md [hunk 3]
```diff
@@ -2,6 +2,40 @@
...
+
+1. Select **Projects** from the sidebar.
+
+2. Click **Create New Project** card or the **+** icon.
+
+3. In the **Basic Info** form, specify details on `Project Name`, `Internal Name` and `Environment Name`.
+
+4. Under the KintoBlocks section, search for the Kintoblock you would like to add. E.g. Microservice Example.
+
+5. Again search for the same KintoBlock and add it to the Project.
```
Comment (Disturbing) [inline docs/features/deploying/manage-projects.md]: Make a separate document for "Searching again and adding multiple kinto blocks".

File: docs/features/deploying/manage-projects.md (hunk 2 above)
Comment (Disturbing) [inline docs/features/deploying/manage-projects.md]: Host names can only be set prior to deploying the block. If you wish to change a blocks name, you must undeploy it first then redeploy, or deploy the same block with a different hostname first, then remove the old one.

**Comment by ajesh-cc** [comment]
Hi Coop, Updated the changes as suggested for the Hostname feature.

**Pull request merged**